common/doc-dev/conf.py

```diff
@@ -25,6 +25,7 @@
 sys.path.insert(0, os.path.abspath(os.path.join(os.pardir, "plugins")))
 
 #import config to solve race conditions between config an mount
```

Comment:
I'm not sure how to deal with this . Does this mean the unused import needs to stay for some reason?

Reply:
How did you find that line? You shouldn't get an error about it because of the comment `pylint: disable-next..`.

Anway you can delete this `import` and the related comment lines.

I am not 100% sure about it. But I checked the git history and couldn't find a good reason to keep that import. I was also able to build the doc via Sphinx without that import.

Reply:
The "pylint: disable-next..." wasn't there, I added it in commit 67506d3. Thats how I found the line.